.github/workflows/test.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install tox tox-gh-actions
+        python -m pip install tox tox-gh-actions pytest
```

Comment:
Is this needed outside of `tox.ini` instead of in the `deps` inside that file?

Reply:
Moved 😁
1f013413bbbeaa03f9c377297c865ac633ceae6a